core/src/main/java/org/infinispan/factories/EmptyConstructorFactory.java

```diff
@@ -98,7 +99,7 @@ else if (componentName.equals(ClusterContainerStats.class.getName()))
       else if (componentName.equals(XSiteCacheMapper.class.getName())) {
          return new XSiteCacheMapper();
       } else if (componentName.equals(XSiteEventsManager.class.getName())) {
-         return new XSiteEventsManagerImpl();
+         return globalConfiguration.isClustered() ? new XSiteEventsManagerImpl() : NoOpXSiteEventManager.INSTANCE;
```

Comment:
Ah! @jabolina reported this to me last night and I was pretty sure I'd fixed it at some point... and I just found it. I'm going to remove this change since it is included in https://github.com/infinispan/infinispan/pull/11375